src/server/routes/requested.js

```diff
@@ -6,8 +6,17 @@ const { unwrap } = require('../../db/util')
 module.exports = router;
 
 router.get('/', async (req, res) => {
+  const status = req.query.status
```

Comment:
@saragw6 should the query param for get be called action or status?

Reply:
i think for now we should call it action (and probably should change the naming for the `setstatus` endpoint for entries). i think i mis-named the action table and it might have been preferable to call them statuses but that's a problem for another time

Reply:
what do you think?

Reply:
yeah action is fine